### DISCUSSION TITLE
Freeze & Reboot, After Hibernate, BSOD  Video DXKernel Fatal Failure

### DISCUSSION THREAD

**Discussion by Sawa-i** [seerge/g-helper#3254] [Q&A]
Hi, writing to just get an idea if anyone else faces this issue OR has anyone resolved.

I have ASUS ROG STRIX G713PV (2023) and using GHelper atm.

I used to get random freezes and reboots, which I resolved like 80% atleast easily by turing off ' Maximum Power Saving'  in:  
[ Windows PowerPlan > Advanced Power Settings (PowerOptions)> PCI Express > Link State Power Management ] 
|
As mentioned in my ROG Post here: [( Solved? ) - Random Freezes and Reboots | ASUS ROG STRIX G17 2023 (G713PV)
](https://rog-forum.asus.com/t5/rog-strix-series/solved-random-freezes-and-reboots-asus-rog-strix-g17-2023-g713pv/td-p/1040270)

-----

But one thing I still face is at times, after using SLEEP / Hibernate.

When I unhibernate, I at times get BSOD Error ( VIDEO DXKRNEL FATAL ERROR ) and reboot.

**Comment by llDoomSlayerll** [comment]
I may be late to the thread but im facing nearly the exact issue like yours on my TUF Dash F15 2022, i have tested everything (4 different SSDs swapping slots, reinstalling Windows on each one of them, DDU'd gpu drivers, etc) and yeah i get eventually get those random freezes or system crashes every 3-7 days, there can be days i can play for 10hrs and no issues, but the next day i play it could crash at any moment. I discovered that Armoury Crate downloads a fined tuned profiles and both performance/turbo got PCIe power management disabled while plugged meanwhile G Helper treats all profiles to maximum energy saving which may cause the SSDs stop responding due to agressive L1/L0 states change, cause whenever i get a crash or system freeze the computer is still working but the SSD storage activity stops blinking which could be that G-Helper does not disable the power management mode at higher system draw profiles (performance/turbo)

**Reply by Sawa-i** [reply]
> I may be late to the thread but im facing nearly the exact issue like yours on my TUF Dash F15 2022, i have tested everything (4 different SSDs swapping slots, reinstalling Windows on each one of them, DDU'd gpu drivers, etc) and yeah i get eventually get those random freezes or system crashes every 3-7 days, there can be days i can play for 10hrs and no issues, but the next day i play it could crash at any moment. I discovered that Armoury Crate downloads a fined tuned profiles and both performance/turbo got PCIe power management disabled while plugged meanwhile G Helper treats all profiles to maximum energy saving which may cause the SSDs stop responding due to agressive L1/L0 states change, cause whenever i get a crash or system freeze the computer is still working but the SSD storage activity stops blinking which could be that G-Helper does not disable the power management mode at higher system draw profiles (performance/turbo)

Hmm, I have a couple of questions:
**Q1)** Since when did you start having the issues? ( Since bought? When bought?)
**Q2)** You mentioned Armoury Crate profiles have PCIe ( link state ) power management disabled on Performance/Turbo... while G-Helper doesn't  Disable "Power Saving", ........ How did you confirm this?
**Q3)** What's your current NVIDIA GameReady Graphic Driver?

------------------

Couple of updates on my end:

1) I'm using G-Helper still because of lower CPU temps compared to Armoury Crate ( hadn't tried re-installing in 6 months)
2) Power Options > PCIe Link State Power Management disabling  brought down my random reboots upto 90%.   Normally I started noticing that if I boot my laptop without plugged in, there are high chances of that BSOD and reboot.
3) Latest Nvidia Graphic GameReady Driver (576.02 ) brought my crashes quite low (only experienced once)......... This could just be me hallucinating atm.
Note: Latest driver has a bug regarding 3rd Party GPU Thermal Sensor detection failing... I'm content as my fans are working fine and Nvidia Overlay can detect that.

|

Plus following are some forums where i've shared my issue and people discussed:

i - https://rog-forum.asus.com/t5/rog-strix-series/solved-random-freezes-and-reboots-asus-rog-strix-g17-2023-g713pv/td-p/1040270 
( CLOSED )

ii- https://rog-forum.asus.com/t5/gaming-notebooks/solved-random-restarts-rog-strix-scar-17-g733pyv-and-other/m-p/1062189#M8772
( CLOSED )

**Reply by llDoomSlayerll** [reply]
Q1) Since 2023, around when i started to use G-Helper, the crashes have happened with 4 different SSDs on both slots and always happens the crash.
Q2) Because when i used G-Helper, it uses Windows Default profile which has both PCIe Power Management to Maximum Saving Mode, it doesnt matters if you apply silent/balanced/turbo on G-Helper, all will have maximum saving mode.
Q3) Im currently using 566.36 due to 576.02 having issues by reading gpu temp sensors, this type of crashes have happened to me with much older gpu drivers so it's not a gpu driver related issue.

1. Yeah same i used G-Helper since 2023 for the same things
2. Very curious, I always plug in my laptop before booting it up, but perhaps a few times i forget to do it, I will consider doing the same thing.
3. I aint using latest drivers until they fix all of that

Also many thanks for quickly responding, i will stay on Armoury Crate for a while.

**Reply by Sawa-i** [reply]
Funny thing, my laptop gone through the BSOD of Freeze and Reboot. It happened during after an hour later of plugging my laptop out and hibernating and unhibernating (while not having it plugged in) 
-> So reboot was kinda expected to happen sometimes as not plugged in boot/unhibernate/unsleep. 

On the bright side, we know (3) Driver update doesn't really resolve this.

|

Rest as mentioned that In Power Options > PCIe Link State PowerManagement, I found the solution of disabling their has brought down these random crashes and reboots by 90%.. ( 10% reboots example was just now )

**Reply by llDoomSlayerll** [reply]
Genuinely thanks for your insight, chatgpt suggested me of disabling PCIe power management due to how i specifically described the freezes happen and any observations worth mentioning (my ssd activity lights stop blinking whenever the system freezes), and i started digging and its not normal to have PCI power management on agressive, specially when you are plugged, most gaming PCs disable that power management by default preventing the GPU or SSD stalling or losing connection whenever they interchange states (L0/L1.1/1.2/P0-P8)).

And then, i realized how armoury crate fined tuned 3 different profiles based on my hardware configuration compared to G-helper treating it like a generic asus gaming laptop. Like i said before, i got an tuf dash f15 2022 model, and if this issue correlates with yours (ROG STRIX 2023) then it could an extremely niche issue.

Could be worth mentioning that a similar setting also impacts the GPU as well, on nvidia app or control panel you can set the gpu to highest performance preventing agressive P0-P8 state changes if your hardware is extremely sensible to that. Let me know if you have tried this.

**Reply by Sawa-i** [reply]
Btw, don't trust ChatGPT. 

If you look at the source of its information, it's the forums posts I posted with labelled answers. It started suggesting disabling PCIe solution after I had posted in the forums.

**Reply by llDoomSlayerll** [reply]
I also asked Gemini and same results with it, if none of these issues fixes the random freezes and system crashes then it could be a hardware issue but there's nothing i could do for that (replace whole motherboard will be brutally expensive might as well get a new laptop) but yeah, will stay on AC for 14-30 days to see if the issues have been fixed and perhaps eventually migrate to G-helper and globally disable PCIe power management. At the end of the day, i remember my laptop working fine during the first months without these shenanigans and decided to try out G-helper due to people on Reddit complaining about AC and being a solid alternative, but why did i even tried to fix something that wasnt broken in the first place.

**Reply by llDoomSlayerll** [reply]
After 15 days of testing i can finally conclude that the issue has been fixed, has been the longest time i haven't had a system freeze and meticulously tested my laptop A LOT (played multiple games +3hrs, internet browsing, video editing, compression/decompression files, etc). So yes, it seems our laptops have a poor way to handle the power management with the SSDs causing these issues, by the way, i strongly suggest disabling erp on BIOS if you have that setting (worth mentioning but perhaps may have no effect at all). Massive thanks for your insights, truly.

Lastly i will still continue monitoring the system for another 15 days, i truly hope G-helper developers add an option to disable Power Management mode per power plan through the app itself.

**Reply by Sawa-i** [reply]
Glad to hear you got to experience your laptop without nuisance.

About Erp، is your off? I can try turning it off next time I have Bsod.

Disabling Power Management through G-Helper might be a good idea... As I've heard couple of people who've had their issue resolved, while there are also others who didn't.

I'll leave it to @seerge to whether have it voted out or what.

**Reply by llDoomSlayerll** [reply]
My bios disables it by default but tried once enabling it and bugged one of my SSDs in the past, but yeah may have an effect at all.

But yeah i strongly suggest G-Helper devs add on the advanced tab to disable Power Management Globally or per profile and adding further explanations that it could improve SSD performance (i tested with crystaldiskmark on both of my SSDs with that setting toggled on/off and there was a very significant difference in fact). And genuinely thank you so much by opening this small thread, you saved all the headaches i had with this laptop over a year lol.

**Reply by Sawa-i** [reply]
> My bios disables it by default but tried once enabling it and bugged one of my SSDs in the past, but yeah may have an effect at all.
> 
> But yeah i strongly suggest G-Helper devs add on the advanced tab to disable Power Management Globally or per profile and adding further explanations that it could improve SSD performance (i tested with crystaldiskmark on both of my SSDs with that setting toggled on/off and there was a very significant difference in fact). And genuinely thank you so much by opening this small thread, you saved all the headaches i had with this laptop over a year lol.

So I checked and it is disabled in my Bios. 
Rest I'm glad to hear you relieved and happy about it, because I personally myself know such suffering... It makes me happy even when someone's suffering is slightly lessened.

**Comment by seerge** [comment]
@llDoomSlayerll 

> But yeah i strongly suggest G-Helper devs add on the advanced tab to disable Power Management Globally or per profile and adding further explanations that it could improve SSD performance (i tested with crystaldiskmark on both of my SSDs with that setting toggled on/off and there was a very significant difference in fact).

How exactly do you disable this "Power Management" ?

**Reply by Sawa-i** [reply]
> @llDoomSlayerll
> 
> > But yeah i strongly suggest G-Helper devs add on the advanced tab to disable Power Management Globally or per profile and adding further explanations that it could improve SSD performance (i tested with crystaldiskmark on both of my SSDs with that setting toggled on/off and there was a very significant difference in fact).
> 
> How exactly do you disable this "Power Management" ?

**Windows Power Plan > Advanced Power Settings > PCI Express > Link State Power Management** 

- On Battery  
- Plugged in 

Whereas the options for each are : Max Power Saving, Min Power Saving, Off.

<img width="991" alt="image" src="https://github.com/user-attachments/assets/a1346498-7ffe-4fda-8bda-a0fee23182cc" />

-----

@llDoomSlayerll , would confirm as well.

**Reply by seerge** [reply]
@Sawa-i ah, that one. 

But since it's visible by default in power plan and needs to be turned off only in exotic troubleshooting cases, I think it's better to keep it there instead of overloading app UI with features that majority doesn't need to tweak :)

I might add it to this advice to the Troubleshooting section tho.

**Reply by Sawa-i** [reply]
To be honest, I also felt the same. But I had also pounced upon many people with similar issues of which at best half of them got resolved through this method, hence was thinking of adding it might be good.

But Yes, your mentioning of this in "Troubleshooting" Section would definitely answer my point.

Btw @seerge, do mention that "Author also mentioned that Unhibernate or Booting while plugged-in had increased stability (i.e. random reboots have more chance when on battery, even more so if unslept/unhibernated)"

**Reply by llDoomSlayerll** [reply]
By the way @Sawa-i, I noticed the system freezed happened to me yesterday but my laptop was sleeping (closed the screen lid) so yeah you were right that this type of issue still happens whenever the laptop enters on hibernate/sleep mode. Mind you this is something that has happened to me since 2022 where on an extremely rare case the laptop wont work if you leave it on sleep/hibernate.

So these are the things i do now

-Installed G-Helper back and disabled PCIe Power Management (both plugged and battery mode)
-Disabled Fast Startup
-Removed sleep option 
-Prevent the laptop to ever go to sleep (Set it to Never) (only allow the display to turn off)

Mind you during those days i heavily tested the laptop A LOT, actually more than ever and no issues whatsoever, but yesterday left the laptop on sleep for like 1-2hrs and when i opened it the system freezed so yeah, issue still occurs but on that scenario.

**Reply by Sawa-i** [reply]
Did you left laptop on sleep while Plugged in?

I can't recall exactly but I might get such issue but during plugged in, it's alot better compared to On Battery.

**Reply by llDoomSlayerll** [reply]
Yeah i left the laptop on sleep while plugged in, i remember it used to happen to me but EXTREMELY RARE like since 2022 but yesterday i left the laptop (closed screen) for 1-2hrs and when i opened it up it freezed. Still during these weeks of testing i used the laptop for +8hrs each day with full demand constantly and 0 issues so the main issue has been fixed but gotta do workarounds of never leaving the laptop on sleep (either afk or closing the screen)
